tinymce-language-selector/plugin.js

```diff
@@ -74,19 +74,28 @@ tinymce.PluginManager.add('language', function(editor) {
       }
     }
     return selectedLang;
-  }
+  };
```

Comment:
We should probably set up a linter for this repo. I'm not sure how that's done. Maybe make a ticket for it?